nextstrain_profiles/nextstrain-open/builds.yaml

```diff
@@ -18,6 +18,14 @@ slack_channel: "#ncov-genbank-updates"
 
 genes: ["ORF1a", "ORF1b", "S", "ORF3a", "E", "M", "ORF6", "ORF7a", "ORF7b", "ORF8", "N", "ORF9b"]
 include_hcov19_prefix: False
+warning: >
+  On Oct 1, 2025, we received an email from GISAID stating that they will no
+  longer be updating the flat file of EpiCoV data they've historically
+  provisioned to Nextstrain since Feb 2020. Updates to GISAID-based analyses are
+  currently on hold. Please use this analysis based on open data instead,
+  updates to which are continuing at their regular weekly cadence. For data to
+  be included in this analysis, please submit these to NCBI via their
+  [SARS-CoV-2 submission portal](https://submit.ncbi.nlm.nih.gov/sarscov2/).
```

Comment:
It's a little strange to jump straight into talking about GISAID on the open dataset. How about starting with something like

"This open (NCBI / GenBank) analysis is now our default view into SARS-CoV-2 evolution. Previously we defaulted to GISAID-sourced data, however starting October 1st, 2025 GISAID indicated they will no longer.... "